### ISSUE TITLE
Recurring Payments: Voiceover Cursor Focus on Payment Form

### ISSUE THREAD

**Issue by donalirl** [Automattic/wp-calypso#41910]
I am opening this up on behalf of a visually-impaired user who uses Apple Voiceover,

#### Steps to reproduce

1. Using a screen reader such as Apple Voiceover, navigate to and tap on the payment button created with a Recurring Payments block.
2. The credit card payment form opens up — attempt to start filling this in.

#### What I expected

I should be able to tap on the payment button, and the cursor should be focused at the top of the page so I can begin swiping right (using a screen reader) to enter fields.

#### What happened instead

The voiceover cursor is not focused on the form so it's not possible to start entering the credit card details into the form.

#### Browser / OS version

Device: iPod Touch
Browser: Safari Latest
Screen reader: Apple Voiceover

#### Screenshot / Video

https://d.pr/i/BIrmeM

#### Context / Source

#p4vt7e-6n-p2

#journey #empathy #livesharing #accessibility #painpoint #user-report

**Comment by donalirl** [comment]
I think this is supposed to be in the Jetpack repo, so closing and reported in /jetpack/issues/15705 instead
